ydr/ydrexport.py

```diff
@@ -798,8 +798,8 @@ def create_joints_xml(armature_obj: bpy.types.Object, auto_calc_bone_tag: bool =
     for pose_bone in armature_obj.pose.bones:
         limit_rot_constraint = get_limit_rot_constraint(pose_bone)
         limit_pos_constraint = get_limit_pos_constraint(pose_bone)
-        bone_tag = pose_bone.bone.bone_properties.tag if auto_calc_bone_tag else calculate_bone_tag(
-            pose_bone.bone.name)
+        bone_tag = calculate_bone_tag(
+            pose_bone.bone.name) if auto_calc_bone_tag else pose_bone.bone.bone_properties.tag
```

Comment:
The other place where `calculate_bone_tag` is used, it also checks for the index to see if it is a root bone, which requires `bone_tag == 0`. Can joints be created on the root bone? If so, it would have the incorrect bone tag here.
https://github.com/Skylumz/Sollumz/blob/aec8a96d6a9921e3ebcafb1248a31a18c3b0a333/ydr/ydrexport.py#L654-L656

Reply:
It shouldn't affect since this fix is meant to be used with manually set boneTags, everything else related to the auto-calculatio of boneTags remains the same.